scripts/common/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+if(DPDK_TARGET)
+   set(TARGET_NAME "dpdk")
+elseif(ES2K_TARGET)
+   set(TARGET_NAME "es2k")
+endif()
+
+configure_file(copy_config_files.sh.in ${CMAKE_INSTALL_PREFIX}/sbin/copy_config_files.sh @ONLY)
```

Comment:
This line looks long. Does it exceed 80 characters?

Reply:
Fixed it